src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/GetInstalledPackagesAsyncTelemetryEvent.cs

```diff
@@ -9,7 +9,7 @@ public sealed class GetInstalledPackagesAsyncTelemetryEvent : DiagnosticTelemetr
     {
         private const string Data = "Data";
 
-        private static readonly string EventName = $"{DiagnosticEventName}/GetInstalledPackagesAsync";
+        private const string EventName = DiagnosticEventName + "/GetInstalledPackagesAsync";
```

Comment:
I applied Roslyn analyzer code fix suggestion which changed the line of code to `private const string EventName = $"{DiagnosticEventName}/GetInstalledPackagesAsync";`. This line of code raised following error message.
`CS8652	The feature 'constant interpolated strings' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.` Hence I ended up with this line of code.